entities/Ingredients.java

```diff
@@ -0,0 +1,53 @@
+package entities;
+
+import constants.Status;
+
+import java.time.Instant;
+
+public class Ingredients extends Base {
+
+    //Attributes
+
+    private String name;
+    private Double quantity;
+    private String unit;
```

Comment:
What is unit?

Reply:
If this is meant to be a measure unit, then please use a enum instead, we should control the measure unit values

Reply:
Yes is a measure unit. so, I'll have to create a constant variable like I did with status, or this can be done within the same entity?

Reply:
You will need to create a new file with the different measure units you are going to support

Reply:
Yes a new file, but this new file should be saved in the same folder we have the Status File?

Reply:
the constant folder

Reply:
Done